DepositRegistrar.sol

```diff
@@ -0,0 +1,306 @@
+// Name registrar using deposits
+
+contract Deed {
+    // The Deed is a contract intended simply to hold ether
+    // It can be controlled only by the registrar and can only send ether back to the owner
+    address public owner;
```

Comment:
Fair enough. Someone desiring that would have to be able to send the bid from a contract that can't be traced back to them too, though, so I'm not sure the gain in anonymity is worth the complexity.